src/main/java/com/epam/lab/payments/model/ApiController.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.lab.payments.model;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class ApiController {
+
+    private final CardRepository cardRepository;
+
+    public ApiController(CardRepository visitsRepository) {
```

Comment:
Don't implement constructors by yourself, use lombok.

Reply:
This file will be deleted and replaced with another controller.